### ISSUE TITLE
IERS B not always loaded into IERS_Auto

### ISSUE THREAD

**Issue by aarchiba** [astropy/astropy#9205]
The IERS B data is not loaded into the IERS_Auto object if new IERS A data is not available, even though the IERS B file is present.

This is demonstrated in PR #9204 .

**Comment by aarchiba** [comment]
In fact what happens is that the IERS_Auto just serves up an unmodified IERS B file as if it were an IERS A file. This means all the _A columns are missing (naturally enough), all the _B columns are missing (which is awkward and unnecessary), and all the unmarked columns are taken straight from the IERS B table. The units of dX_... and dY_... are different in IERS B than IERS A (arcseconds versus milliarcseconds) but this is probably not a serious problem as long as people never take the .value without a .to() first. (The code in iers.py used to do this, so it's a realistic concern.)

**Comment by pllim** [comment]
cc @taldcroft and @mhvk

**Comment by ayshih** [comment]
I think this particular issue is largely moot now that both IERS-A and IERS-B are bundled via `astropy-iers-data` (#14819).  There are still lingering questions about merging in IERS-B data (e.g., #9227 and #13494), but they are probably better discussed in those issues.

**Comment by mhvk** [comment]
I agree that this is moot given `astropy-iers-data` - let's close it.
